test/acceptance/test/utils.go

```diff
@@ -216,14 +216,18 @@ func ResetOrCreateClusterWithName(namespace string, deleteWegoRuntime bool, clus
 	return clusterName, nil
 }
 
-func initAndCreateEmptyRepo(appRepoName string, isPrivateRepo bool) string {
+func initAndCreateEmptyRepoWithMainDefaultBranch(appRepoName string, isPrivateRepo bool) string {
+	return initAndCreateEmptyRepo(appRepoName, DEFAULT_BRANCH_NAME, isPrivateRepo)
```

Comment:
Seems odd to wrap this when it's only saving one argument that would already be a `const`. I guess it is saving us from passing `DEFAULT_BRANCH_NAME` in everywhere?

Reply:
Well, I needed to be able to call it in _one_ test with a different branch and thought it would be unfortunate to have to pass the same argument in _every call but one_ (and then one would have to go look to determine what branch was being passed).